scripts/withdraw.js

```diff
@@ -81,7 +81,7 @@ const getAmount = async function(bracketAddress, tokenAddress, exchange) {
 module.exports = async callback => {
   try {
     const masterSafePromise = getSafe(argv.masterSafe, artifacts)
```

Comment:
I think it would be cleaner to await the promise here already and store the variable the masterSafe. Or do we have any gains by storing only the promise?

Reply:
The Safe is needed only at the very end, so loading it in the background from the start is a little efficiency gain. I never measured the time this saves, but it should be a the length of a full network request.